x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation.cy.ts

```diff
@@ -92,32 +85,6 @@ describe.skip('Timelines', { tags: ['@ess', '@serverless'] }, (): void => {
     );
   });
 
-  it('should create a timeline by clicking untitled timeline from bottom bar', () => {
```

Comment:
Explanation of removal of this test

1. I do not see the point of this test.
2. It does not make sense to test `pin` here and is also tested in jest tests.

Reply:
Is this the only place where we test pinning of events in Cypress? If so, we might want to add a real integration test to this.